packages/grid_client/src/high_level/farmerbot.ts

```diff
@@ -43,6 +43,7 @@ class FarmerBotFindNodeModel {
   @Expose() @IsOptional() @IsBoolean() public_config?: boolean;
   @Expose() @IsOptional() @Min(0) public_ips?: number;
   @Expose() @IsOptional() @IsBoolean() certified?: boolean;
+  @Expose() @IsOptional() @Min(0) has_gpus?: number;
```

Comment:
is it a bool or int?

Reply:
This int based on Brandon's implementation [here](https://github.com/threefoldtech/farmerbot/pull/46/files#diff-52d02683e7c1d33c03f34ad7bf9bdb9d7e9dca9eb67162dd66a33a933008d329R84)